frontend/src/pages/projects/dataConnections/const.ts

```diff
@@ -60,10 +61,10 @@ export const EMPTY_AWS_SECRET_DATA: AWSDataEntry = [
   },
   {
     key: AWS_KEYS.S3_ENDPOINT,
-    value: 'https://s3.amazonaws.com/',
+    value: '',
   },
   {
     key: AWS_KEYS.DEFAULT_REGION,
-    value: 'us-east-1',
+    value: '',
```

Comment:
value is not needed

Reply:
value is a required field, I've removed the value's value though.

Reply:
yep, you are right. i was looking at the wrong object